integration/generation_terraform_test.go

```diff
@@ -0,0 +1,149 @@
+//go:build generation || (!windows && generation)
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/hashicorp/go-version"
+	"github.com/hashicorp/hc-install/product"
+	"github.com/hashicorp/hc-install/releases"
+	"github.com/lacework/go-sdk/cli/cmd"
```

Comment:
ugh...... We have never done this..... 😟 I don't know if I like the way we are importing the CLI.